resources/assets/fontello/css/fontello-ie7-codes.css

```diff
@@ -2,4 +2,13 @@
 .icon-user { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe800;&nbsp;'); }
```

Comment:
This should not be in your repository. Can you not add this to some package manager?

Reply:
We could technically add a package like Fontawesome through `npm`, but that would increase the size of the webpage even more. If you would prefer that, we could replace all fontello-icons with Fontawesome ones.

(Fontello allows for icons from several packages to be selected and imported. It's not a package, sadly. Fontawesome takes about half a MB, I think)

Reply:
Hmm, you do not necessarily have to swap this out for Fontawesome but in general having dependencies in your repository is quite a bad idea. Although I have never done it, I believe you can also use Composer to download packages straight from Github.

Or you could indeed switch to Font-awesome, in which case your webpage size wouldn't increase as much as you think. You can just include whatever icons you want, e.g. see https://fontawesome.com/how-to-use/on-the-web/using-with/vuejs

Reply:
Unfortunately using the `Vue.js` is not really an option, since we do not use Vue globally. We could of course do that, but that seems overkill for simple icons. I'll take another look at the options later, will finish the other fixes first.

Reply:
Alright, I've just replaced Fontello with Fontawesome, adding 1MB+ of files. I'll check if I cannot find a more modular way like the link you gave.